src/newgrf.cpp

```diff
@@ -5131,14 +5131,35 @@ static void SkipAct1(ByteReader &buf)
 	GrfMsg(3, "SkipAct1: Skipping {} sprites", _cur.skip_sprites);
 }
 
+using CachedCallback = std::pair<uint16_t, SpriteGroupID>;
+static std::vector<CachedCallback> _cached_callback_groups; ///< Sorted list of cached callback result spritegroups.
+
+static const SpriteGroup *GetCallbackResultGroup(uint16_t value)
+{
+	/* Old style callback results (only valid for version < 8) have the highest byte 0xFF so signify it is a callback result.
```

Comment:
Should it have been 'to signify'?
```suggestion
	/* Old style callback results (only valid for version < 8) have the highest byte 0xFF to signify it is a callback result.
```